t/01-customers.t

```diff
@@ -18,7 +18,7 @@ subtest 'basic stuff' => sub {
 };
 
 subtest 'create customer with no data' => sub {
-    my $cust = stripe->create_customer;
+    my $cust = stripe->create_customer({});
```

Comment:
Actually, the error is legit:
```perl
use Method::Signatures;

func foo(Maybe[HashRef] $x, :$foo) {
    say 'hi';
}

foo();
```

    In call to main::foo(), missing required argument $x at - line 10.